Use the system python3 instead of the hermetic cpython3 interpreter, which
is not shipped in the tarball and is not needed when building against system
tools.

Credit: https://github.com/ungoogled-software/ungoogled-chromium/pull/3946

--- a/.gn
+++ b/.gn
@@ -9,7 +9,7 @@
 buildconfig = "//build/config/BUILDCONFIG.gn"

 # The hermetic Python interpreter to use by default.
-script_executable = "//third_party/cpython3/host/bin/python3"
+script_executable = "python3"

 # These arguments override the default values for items in a declare_args
 # block. "gn args" in turn can override these.
